packages/main/src/Table.js

```diff
@@ -541,6 +549,103 @@ class Table extends UI5Element {
 		if (isAltUp || isDownAlt(event)) {
 			return this._handleArrowAlt(isAltUp, event.target);
 		}
+
+		if ((isUpShift(event) || isDownShift(event)) && this.isMultiSelect) {
+			this._handleArrowNav(event);
+		}
+
+		if (isHomeCtrl(event)) {
+			this._itemNavigation._handleHome(event);
```

Comment:
Scrolling should be prevented on `CTRL`+`HOME`/`END`:

```suggestion
			event.preventDefault();
			this._itemNavigation._handleHome(event);
```